surfactant/cmd/merge.py

```diff
@@ -42,10 +79,19 @@ def merge_command(input_sboms, sbom_outfile, config_file, output_format, input_f
     config = None
     if config_file:
         config = json.load(config_file)
-    merge(sboms, sbom_outfile, config, output_writer)
-
-
-def merge(input_sboms, sbom_outfile, config, output_writer):
+    merge(sboms, sbom_outfile, config, output_writer, system_uuid, system_relationship, add_system)
+
+
+# pylint: disable-next=too-many-positional-arguments
+def merge(
+    input_sboms,
+    sbom_outfile,
+    config,
+    output_writer,
+    system_uuid=None,
+    system_relationship="Contains",
```

Comment:
default is already specified in the click option, do we need to specify it here too?

Reply:
Added a comment reminder to keep in-sync with click default -- I think the only time this default value would be used is if someone is using Surfactant as a library and directly calling this merge function from within their own Python scripts.